README.md

```diff
@@ -1,9 +1,6 @@
 GraphQL for .NET - Subscription Transport WebSockets
 ====================================================
 
->WARNING: not tested in heavy production use! That said if you are using this in production
```

Comment:
I thought to remove this warning but did not. Do you think it's time?

Reply:
We use it in production but are slowly migrating things onto graphql, so it's not getting massive load. I dunno, I feel like it's time. There's enough of us who are I'd say.